packages/frontend/scss/message/_message-detail.scss

```diff
@@ -3,10 +3,7 @@
 .module-message-detail {
   margin-top: -20px;
 
-  .bp4-callout {
-    max-height: 50vh;
-    overflow: auto;
-  }
+  max-height: 55vh;
```

Comment:
This is weird. `max-height` is set, but then the element overflows anyway, which I think results in padding-bottom not being respected. Why do we apply it to the parent element now, instead of the `callout`?

![image](https://github.com/user-attachments/assets/ef2b5b21-fdd1-4a27-bbb9-2809700f806b)